packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1764,28 +1764,14 @@ class OpacityLayer extends ContainerLayer {
   set alpha(int? value) {
     assert(value != null);
```

Comment:
No range assert here.

Reply:
That should be addressed in a separate PR.